content/projects/bash-for-bootcamp/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+_db_id: 394
+content_type: project
+flavours:
+- none
+submission_type: link
+ready: true
+tags:
+- bash
+title: Beginning Bash
+---
+
+In order to interact properly with Git, you'll need to know a little bit about "the command line".
+
+The first step is to open the command line or terminal on your device:
+
+- If you are running Android, then open your termux app
+- If you are running Windows, then
+  - open your git shell OR
+  - using [linux subsystem for windows](https://itsfoss.com/install-bash-on-windows/), just open your linux terminal
+- If you are using mac or linux, jkust open your terminal
+
+If this is the first time you've ever opened such a program, DON'T PANIC! You'll get used to this thing.
+
+Now you'll need to learn a few basic commands to get by:
+
+http://korflab.ucdavis.edu/bootcamp.html
+
+Make sure you understand everything up to step 14. NB: Completing and understanding up to step 14 is essential for successful completion of this project.
+
+Later on, if you get into Umuzi, you'll be expected to know all that stuff. But for the bootcamp, just the basics are ok.
+
+#### Submission Instructions
+
+In this project you will be submitting a link to a google document. 
+
+Please copy and paste the questions below including the question number and the full question into your google doc and answer the question on a new line below, use additional new lines and spacing to make the submission easy to read and assess.
+
+Ensure you update the sharing permissions on your google doc to allow anyone with the link to view it.
+
+#### Questions
+
+1. What is the purpose of the `ls` command?
+2. What does the command `ls /` output?
+3. What does the `pwd` command do?
+4. When would you make use of the `pwd` command?
+5. What command would you use to make a new directory?
+6. What command would you use to navigate into a new directory?
+7. What command would you use to ‘upwards’ one level in the directory hierarchy?
```

Comment:
Some words might be missing. 
Eg: 7. What command would you use to MOVE ‘upwards’ one level in the directory hierarchy?